airbase/airbase.py

```diff
@@ -5,9 +5,9 @@
 from datetime import datetime
 from pathlib import Path
 
-if sys.version_info >= (3, 8):
+if sys.version_info >= (3, 8):  # pragma: no cover
```

Comment:
Why do we need all these new pragmas?

Reply:
They count against the coverage, as it each test run goes only trough one of the if branches. One could combine the coverage reports from different python versions, but I find the set up cumbersome.

Please let me know if you want me to remove them.

Reply:
No it's fine to leave them, I just wasn't sure if you added this because of the changes or of it was just housekeeping